Makefile

```diff
@@ -85,11 +85,20 @@ push-webhook-image: webhook-image
 	}
 
 # Build the docker image for the core CSI driver.
-image:
+image: init-buildx
 		{                                                                   \
 		set -e ;                                                            \
-		for i in $(STAGINGVERSION) ;                                        \
-			do docker build --build-arg DRIVERBINARY=$(DRIVERBINARY) --build-arg TAG=$(STAGINGVERSION) -t $(STAGINGIMAGE):$${i} .; \
+		for i in $(STAGINGVERSION) ; do                                     \
+			docker buildx build \
+				--platform linux/amd64 \
+				--build-arg TAG=$(STAGINGVERSION) \
+				--build-arg VERSION=$(STAGINGVERSION) \
+				--build-arg STAGINGVERSION=$(STAGINGVERSION) \
+				--build-arg BUILDPLATFORM=linux/amd64 \
+				--build-arg TARGETPLATFORM=linux/amd64 \
+				--build-arg ARCH=amd64 \
```

Comment:
It looks like ARCH, TAG, and VERSION aren't actually used as build args in the Dockerfile? Should probably remove them to keep things clean.

We don't use them in the pdcsi driver any more either, probably build-image in louhi should be cleaned up but that's a separate task.

Reply:
DONE.